libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -176,6 +176,13 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS",  15, AP_ADSB, _options, 0),
 
+    // @Param: MODE3_ONLY
```

Comment:
Why didn't you use an option bit for this?

Reply:
Hmm. yeah, thats a good idea

Reply:
I've updated this PR to be an options bit. Thanks for the idea!